docs/articles/documentation/test-api/selecting-page-elements/examples-of-working-with-dom-elements.md

```diff
@@ -81,7 +102,92 @@ test('My test', async t => {
 });
 ```
 
-## Checking if an Element is Available
+## Access Child Nodes in the DOM Hierarchy
+
+The [selector API](selectors/functional-style-selectors.md) allows you to filter matching elements and search through adjacent elements in the DOM tree.
+
+Selector API contains two types methods:
+
+* methods that enumerate all node types
+* methods that enumerate DOM elements only
+
+<!-- markdownlint-disable MD033 -->
+Methods                                                | Enumerated Nodes
+------------------------------------------------------ | -------
+[filter(function)](selectors/functional-style-selectors.md#filter)<br/>[find](selectors/functional-style-selectors.md#find)<br/>[parent](selectors/functional-style-selectors.md#parent) | All nodes
+[nth](selectors/functional-style-selectors.md#nth)<br/>[withText](selectors/functional-style-selectors.md#withtext)<br/>[withExactText](selectors/functional-style-selectors.md#withexacttext)<br/>[withAttribute](selectors/functional-style-selectors.md#withattribute)<br/>[filter(string)](selectors/functional-style-selectors.md#filter)<br/>[filterVisible](selectors/functional-style-selectors.md#filtervisible)<br/>[filterHidden](selectors/functional-style-selectors.md#filterhidden)<br/>[child](selectors/functional-style-selectors.md#child)<br/>[sibling](selectors/functional-style-selectors.md#sibling)<br/>[nextSibling](selectors/functional-style-selectors.md#nextsibling)<br/>[prevSibling](selectors/functional-style-selectors.md#prevsibling) | Elements only
+<!-- markdownlint-enable MD033 -->
+
+The following example illustrates the difference between these methods and shows how to get a child text node for a given parent element.
+
+Consider the following *example.html* page:
+
+```js
+<!DOCTYPE html>
+<html>
+    <body>
+        This is my tested page. <!--This is the first child node of <body>-->
+        <p>My first paragraph.</p>
+        <p>My second paragraph.</p>
+    </body>
+</html>
+```
+
+Let's write a test that verifies text content of the body's first child node (*'This is my tested page'*).
+
+To select this node, use the [filter(function)](selectors/functional-style-selectors.md#filter) method that enumerates all nodes. Compare it with the [nth](selectors/functional-style-selectors.md#nth) method that skips the text node and returns the `<p>` element.
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `My Fixture`
+    .page `example.html`;
+
+const body              = Selector('body');
+const firstChildElement = body.nth(0);                   // <p>
```

Comment:
Don't get it. 
The `nth` method returns the first element from Selector's result set. `Selector('body')` returns only one element because it's body, so we don't need `nth(0)` at all. In this case `Selector('body')` and `Selector('body').nth(0)` have the same result -> body element.
`To select this node, use the [filter(function)](selectors/functional-style-selectors.md#filter) method that enumerates all nodes.` But in the example we have  the `find` method 😮

Reply:
Oh...

Reply:
Now I've run it - seems okay this way.